modules/retail/pages/retail-formulas-intro.adoc

```diff
@@ -3,20 +3,20 @@
 
 
 Formulas are pre-written Salt states, that are used to configure your {smr} installation.
-
 This section lists the primary formulas shipped with {smr} and their configuration options.
 
 All the formulas in this section must be accurately configured for your {smr} installation to function correctly.
 If you are unsure of the correct formula configuration details, run the [command]``retail_branch_init`` command before you begin to create the recommended formula configuration.
 You can then manually edit the formulas as required.
 
 
-.State and formula name collisions
+.State and Formula Name Collisions
 [IMPORTANT]
 ====
 If a formula uses the same name as an existing Salt state, the two names will collide, and could result in the formula being used instead of the state.
 Always check the names of states and formulas to avoid name collisions.
 ====
 
 Most formulas can be updated using the {susemgr} {webui}.
-Once you have made changes to your formula, ensure you apply the highstate to propagate your changes to the appropriate services.
+When you have made changes to your formula, ensure you apply the highstate.
+With the highstate you propagate your changes to the appropriate services.
```

Comment:
```suggestion
The highstate propagates your changes to the appropriate services.
```

This sentence is possibly redundant, though?

Reply:
Yes, but here in the intro it's ok to give more background info.